### ISSUE TITLE
nugetRestore.proj warning about non-existing project

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#382]
```
What steps will reproduce the problem?

1. Clone repo (ATR revision 0d650a436f7f    )
2. Run build.cmd from the root of the local clone

The following warning is seen that needs investigation:

"C:\ELMAH\nugetRestore.proj" (default target) (1) ->
(RestoreNuGetPackages target) ->
  EXEC : warning : Project file C:\ELMAH\http://localhost:51296 cannot be found. [C:\ELMAH\nugetRestore.proj]

```

Original issue reported on code.google.com by `azizatif` on 4 Mar 2015 at 7:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `jamesdriscoll71` on 9 Mar 2015 at 11:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in https://code.google.com/r/jamesdriscoll71-demo-project-issues
```

Original comment by `jamesdriscoll71` on 10 Mar 2015 at 12:05
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Changing Status to Started as fix has not yet been pulled into the main repo.
```

Original comment by `azizatif` on 10 Mar 2015 at 6:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 12 Mar 2015 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2454e3908c64.
```

Original comment by `azizatif` on 12 Mar 2015 at 6:25
- Changed state: **Fixed**
